app/frontend/components/pages/sendAda/sendAdaPage.tsx

```diff
@@ -165,7 +167,7 @@ const SendAdaPage = ({
 
   const adaAsset: DropdownAssetItem = {
     type: AssetFamily.ADA,
-    policyId: null,
+    policyId: '',
```

Comment:
non-null policyId doesn't make sense for ADA asse. The type of DropdownAssetItem should rather be fixed to have policyId as null for this case and as not-null for the case when the type is AssetFamily.TOKEN, leveraging possibly type union